### ISSUE TITLE
Lens generation issues

### ISSUE THREAD

**Issue by eed3si9n** [eed3si9n/scalaxb#339]
Bumping up the Scalaz version to 7.1.3 causes tests to fail:

```
[trace] Stack trace suppressed: run last integration/test:test for the full output.
[error] Could not run test IgnoreUnknownPurchaseOrderTest: java.lang.IncompatibleClassChangeError: Found class scalaz.syntax.FunctorOps, but interface was expected
```

Also https://groups.google.com/d/msg/scalaxb/t4YfppEtL9s/kH5uOZCpdFEJ

> I have started using them but I am encountering a problem:
>
> ScalaXB generate Seq collection and scalaz is not compatible with them.

At this point I am thinking about making a branch for this feature, and gutting it out of mater.

**Comment by eed3si9n** [comment]
https://github.com/eed3si9n/scalaxb/tree/topic/lens

**Comment by schrepfler** [comment]
Does this branch rely on the varargs to Seq capability introduces in 1.4?

**Comment by eed3si9n** [comment]
@schrepfler Yes. no-varargs was introduced originally as part of #292, but I kept it around for 1.4.0 since I've been planning to do that for scalaxb 20XX.
